app/src/main/java/com/henagle/www/kanban/MainActivity.java

```diff
@@ -1,133 +1,104 @@
 package com.henagle.www.kanban;
 
-import android.content.Context;
 import android.content.DialogInterface;
 import android.os.Bundle;
-import android.support.annotation.NonNull;
-import android.support.design.widget.BottomNavigationView;
+import android.support.v4.view.PagerAdapter;
+import android.support.v4.view.ViewPager;
 import android.support.v7.app.AlertDialog;
 import android.support.v7.app.AppCompatActivity;
 import android.support.v7.widget.Toolbar;
-import android.view.LayoutInflater;
 import android.view.Menu;
 import android.view.MenuItem;
 import android.view.View;
 import android.view.ViewGroup;
-import android.widget.BaseAdapter;
 import android.widget.EditText;
 import android.widget.LinearLayout;
-import android.widget.ListView;
 import android.widget.RelativeLayout;
-import android.widget.TextView;
 
 import com.henagle.www.kanban.model.Card;
-import com.henagle.www.kanban.model.CardSerializer;
+import com.henagle.www.kanban.model.DeckSerializer;
+import com.henagle.www.kanban.model.Deck;
 
 import java.util.HashMap;
-import java.util.List;
 import java.util.Map;
 
 public class MainActivity extends AppCompatActivity {
 
-    private ListView list;
-    private Map<Tab,List<Card>> tabLists;
-    private Map<Tab, CardListAdapter> adapters;
-    private Tab currentTab = Tab.TODO;
```

Comment:
I replaced `tabLists` with `decks`, opting to use the `Deck` object instead of a `List<Card>` object. With the ViewPager, there is no need for three separate adapters - just one `ViewPagerAdapter` that handles multiple pages (`DeckFragment`s, in this case).